tests/test_client_ws_functional.py

```diff
@@ -685,6 +685,36 @@ async def handler(request):
     assert resp.close_code is WSCloseCode.ABNORMAL_CLOSURE
 
 
+async def test_heartbeat_no_pong_concurrent_receive(aiohttp_client: Any) -> None:
+    ping_received = False
+
+    async def handler(request):
+        nonlocal ping_received
+        ws = web.WebSocketResponse(autoping=False)
+        await ws.prepare(request)
+        msg = await ws.receive()
+        if msg.type == aiohttp.WSMsgType.PING:
```

Comment:
```suggestion
        if msg.type is aiohttp.WSMsgType.PING:
```

Reply:
The `test_heartbeat_no_pong` uses an equals sign and I would keep this for consistency among the tests.

Reply:
Enums are singletons so an identity check should be used. The other test should be updated as well.

Reply:
fixed